sdks/python/run_pylint.sh

```diff
@@ -39,14 +39,19 @@ EXCLUDED_GENERATED_FILES=(
 
 FILES_TO_IGNORE=""
 for file in "${EXCLUDED_GENERATED_FILES[@]}"; do
-  if [[ $FILES_TO_IGNORE ]]; then
-    FILES_TO_IGNORE="$FILES_TO_IGNORE, "
+  if test -z "$FILES_TO_IGNORE"
+  then FILES_TO_IGNORE="$(basename $file)"
```

Comment:
OK.
However, I think the non-indented `then` and `else` is more readable in bash in this case, i.e., you should write either:

    if ...
    then single command
    else single command
    fi

or for multiple commands:

    if ...; then
       indented commands
       more commands
    else
       indented command
       more commands
    fi